.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,20 @@
+name: Build docker
+
+on:
+  push:
+  schedule:
+    - cron: "0 12 * * 1"
```

Comment:
What's the reason to build on a cron?

Reply:
so we know it always works

Reply:
Anything that *can* be different between two runs if the code doesn't change?